src/main/java/org/sefglobal/scholarx/util/ProgramUtil.java

```diff
@@ -126,16 +126,16 @@ public void sendMenteeSelectionEmails(long id, Optional<Program> program) throws
                     "It is with much pleasure we inform you that we have completed the first round of mentor-mentee matching.<br />" +
                     "There are a few points that we would like to share with you with regard to the mentee applications.<br />"+
                     "<ul>"+
```

Comment:
@gajindubandara Did you check the other templates for spacing issues as well?

Reply:
No! I'll check the others as well.